tsup.config.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+
+const tsup = require("tsup");
+const { umdWrapper } = require("esbuild-plugin-umd-wrapper");
+
+module.exports = tsup.defineConfig([
+  {
+    entry: {
+      "peggy.min": "./lib/peg.js",
+    },
+    format: ["umd"],
+    dts: true,
+    platform: "browser",
+    minify: true,
+    outDir: "browser",
+    target: "es5",
+    bundle: true,
+    clean: true,
+    splitting: false,
+    noExternal: [/./],
+    esbuildPlugins: [
+      umdWrapper({ libraryName: "peggy", external: [] }),
+    ],
+  },
+  {
+    entry: {
+      "test-bundle.min": "test/all.js",
+    },
+    format: ["umd"],
+    platform: "browser",
+    minify: true,
+    outDir: "docs/js",
+    bundle: true,
+    clean: false,
+    splitting: false,
+    external: ["mocha", "chai"],
```

Comment:
note: do not add peggy here.  The tests use require's deep into the project structure, so the tests and all of peggy have to be bundled together.

Reply:
The benchmarks serve as spot checks of the generated code now.